docs/rounding.md

```diff
@@ -0,0 +1,79 @@
+## Rounding a ZonedDateTime
+
+(Only supported in Julia version 0.5 and later.)
+
+Rounding operations (`floor`, `ceil`, and `round`) on `ZonedDateTime`s are performed in a
+[similar manner to `DateTime`](http://julia.readthedocs.org/en/latest/manual/dates/#rounding)
+and should generally behave as expected. When `VariableTimeZone` transitions are involved,
+however, unexpected behaviour may be encountered.
+
+Instead of performing rounding operations on the `ZonedDateTime`'s internal UTC `DateTime`,
+which would be computationally less expensive, rounding is done in the local time zone.
+This ensures that rounding behaves as expected and is maximally meaningful.
+
+If rounding were done in UTC, consider how rounding to the nearest day would be resolved for
+non-UTC time zones: the result would be 00:00 UTC, which wouldn't be midnight local time.
+Similarly, when rounding to the nearest hour in `Australia/Eucla (UTC+08:45)`, the result
+wouldn't be on the hour in the local time zone.
+
+### Rounding to a TimePeriod
+
+When the target resolution is a `TimePeriod` the likelihood of encountering an ambiguous or
+non-existent time (due to daylight saving time transitions) is increased. To resolve this
+issue, rounding a `ZonedDateTime` with a `VariableTimeZone` to a `TimePeriod` uses the
+`DateTime` value in the appropriate `FixedTimeZone`, then reconverts it to a `ZonedDateTime`
+in the appropriate `VariableTimeZone` afterward.
```

Comment:
Leave things how they are for now. Maybe be nice to add in a simple: "See below for examples"